src/Plug.vala

```diff
@@ -237,6 +245,9 @@ public class DateTime.Plug : Switchboard.Plug {
 
             network_time_switch.active = datetime1.NTP;
             change_tz (datetime1.Timezone);
+
+            var week_number_settings = new GLib.Settings ("org.pantheon.desktop.wingpanel.indicators.datetime");
```

Comment:
I think, before using these schema, you should ensure that it exists so that the plug won't crash on systems without the date time indicator.